raiden/utils/typing.py

```diff
@@ -74,6 +74,12 @@ def typecheck(value: Any, expected: Union[Type, Tuple[Type, ...]]) -> None:
 
 AddressHex = HexAddress
 
+T_UserID = str
+UserID = NewType("UserID", T_UserID)
+
+T_AddressMetadata = Dict[str, Union[UserID, "PeerCapabilities"]]
```

Comment:
PeerCapabilities without " ?

Reply:
This is a forward reference, because PeerCapabilities is defined below.
Another option would be to define the AddressMetadata later, which would probably be  better.